cmd/queue/main.go

```diff
@@ -322,8 +333,51 @@ func main() {
 		logger.Fatalw("Failed to parse localhost URL", zap.Error(err))
 	}
 
+	queueProxyL3 := fmt.Sprintf("%s:%d", env.ServingPod, networking.ServiceHTTPPort)
+	zipkinEndpoint, err := zipkin.NewEndpoint(env.ServingPod, queueProxyL3)
+
+	if err != nil {
+		logger.Error("Unable to create tracing endpoint")
+		return
+	}
+
+	oct := tracing.NewOpenCensusTracer(
+		tracing.WithZipkinExporter(tracing.CreateZipkinReporter, zipkinEndpoint),
```

Comment:
What's the significance of `zipkinEndpoint` here? Is it only used to report where the spans are coming from?

Reply:
We need the exporter to export traces to zipkin & the zipkinEndpoint is the endpoint where zipkin will be receiving the traces.

In `config-tracing.yaml` it will be something like:
`zipkin-endpoint: "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans"`